doc/modules/ROOT/pages/index.adoc

```diff
@@ -99,18 +99,19 @@ public WebFilter webFilter(HazelcastInstance hazelcastInstance) {
 }
 ----
 
-This way, we defined Hazelcast configuration and used the embedded Hazelcast instance to enable session replication.
+This way, we defined a Hazelcast configuration and used an embedded Hazelcast instance to enable session replication.
 
 == Starting the Application
 
 To start two replicas of the application, run the following commands.
 
+[source,bash]
 ----
-mvn spring-boot:run
-SERVER_PORT=8081 mvn spring-boot:run
+$ mvn spring-boot:run
```

Comment:
The same comment as in the other guide review, I'd skip `$` for one-liners, because it makes hard to copy the command and paste into your shell.

Reply:
I agree, removed.